tests/test_create_conference.py

```diff
@@ -247,6 +247,61 @@ def test_create_invitations(self, client, openreview_client):
         client.post_invitation(invitation)
         assert client.get_invitation('openreview.net/-/paper')
 
+        # Test inclusion invitation
+        edge_reply = {
+            "readers": {
+                "values-copied": [
+                    "DEF.cc",
+                    "{signatures}"
+                ]
+            },
+            "writers": {
+                "values-copied": [
+                    "NeurIPS.cc/2022/Conference",
+                    "{signatures}"
+                ]
+            },
+            "signatures": {
+                "values-regex": "~.*"
+            },
+            "content": {
+                "head": {
+                    "type": "Note"
+                },
+                "tail": {
+                    "type": "Profile"
+                },
+                "label": {
+                    "value-radio": [
+                        "Include"
+                    ],
+                    "required": True
+                }
+            }
+        }
+
+        invitation = openreview.Invitation(
```

Comment:
instead of creating the invitation, could you set the flag in the conference object? 

https://github.com/openreview/openreview-py/blob/master/openreview/venue_request/venue_request.py#L1139

'Yes' means the inclusion is on, otherwise the invitation os being use to exclude. Be aware that we use the same invitation id in both cases.